src/components/status-check.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Box, Flex, space, themeColor, color } from '@blockstack/ui';
+import { border } from '@common/utils';
+import { Link } from '@components/mdx';
+import { Text } from '@components/typography';
+import fetch from 'isomorphic-unfetch';
+import { CheckmarkCircleIcon, ExclamationMarkCircleIcon } from '@blockstack/ui';
+
+export const StatusCheck: React.FC<any> = () => {
+  const [status, setStatus] = React.useState(false);
+  const STATUS_CHECKER_URL = 'https://status-checker-blockstack.herokuapp.com';
```

Comment:
Should this be `https://status.test-blockstack.com`?

Reply:
yes! with @yknl's update I can now use this URL instead :)